omnigibson/robots/fetch.py

```diff
@@ -388,6 +392,52 @@ def disabled_collision_pairs(self):
             ["torso_lift_link", "shoulder_lift_link"],
             ["torso_lift_link", "torso_fixed_link"],
         ]
+    
+    @property
+    def primitive_disabled_collision_pairs(self):
+        """
+        Returns:
+            Array of arrays: Disabled collision pairs for necessary for primitives. Including these in disabled_collision_pairs
+            throws an error. When bug is fixed, these can be merged with disabled_collision_pairs.
+        """
+        return [
+            ["torso_lift_link", "shoulder_lift_link"],
+            ["torso_lift_link", "torso_fixed_link"],
+            ["torso_lift_link", "estop_link"],
+            ["base_link", "laser_link"],
+            ["base_link", "torso_fixed_link"],
+            ["base_link", "l_wheel_link"],
+            ["base_link", "r_wheel_link"],
+            ["base_link", "estop_link"],
+            ["torso_lift_link", "shoulder_pan_link"],
+            ["torso_lift_link", "head_pan_link"],
+            ["head_pan_link", "head_tilt_link"],
+            ["shoulder_pan_link", "shoulder_lift_link"],
+            ["shoulder_lift_link", "upperarm_roll_link"],
+            ["upperarm_roll_link", "elbow_flex_link"],
+            ["elbow_flex_link", "forearm_roll_link"],
+            ["forearm_roll_link", "wrist_flex_link"],
+            ["wrist_flex_link", "wrist_roll_link"],
+            ["wrist_roll_link", "gripper_link"],
+        ]
+
+    @property
+    def manipulation_link_names(self):
```

Comment:
what is this?

Reply:
This is the list of links that we used for manipulation. We have this list so that we know which links need to be moved for each robot to set it to a new joint position, which is needed for FK in our arm motion planner.

Reply:
This might not be necessary actually - we can do FK on all joints. Am investigating.

Reply:
So it looks like we'll stick with this for now but have a plan for getting rid of these really soon with some automated descriptors.